teacher.rb

```diff
@@ -0,0 +1,15 @@
+# Teacher class that extends Person
+require_relative 'person'
+
+class Teacher
```

Comment:
Great job @Mukumbuta 🤝

- [ ] Kindly make sure the Teacher class inherits from Person. You may follow this [link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_principles_examples.md#inheritance) for more on inheritance.